stress_test/tiny_test.py

```diff
@@ -0,0 +1,36 @@
+from uuid import uuid4 as uuid
```

Comment:
this whole file was moved... should no longer stay in tests

Reply:
What is its purpose anyway?